src/Common/ode_utils.jl

```diff
@@ -133,4 +148,13 @@ function initialise_aux(FT, ip, common_params, thermo_params, air_params, activa
         Stats = Stats,
         TS = TS,
     )
+    if psNM
```

Comment:
Initialise the pdists in here

Reply:
Also add a struct for `cloudy_params` here to store all of the velocity, kernels, etc. stuff -- make them